app/config/services/cms.xml

```diff
@@ -8,24 +8,45 @@
         <service id="League\CommonMark\CommonMarkConverter" />
 
         <!-- Content -->
-        <service id="app.content.media_factory" class="AppBundle\Content\MediaFactory" />
+        <prototype namespace="AppBundle\Content\" resource="../../../src/Content" />
 
-        <service id="app.content.home_block_factory" class="AppBundle\Content\HomeBlockFactory" />
+        <service id="app.content.media_factory" alias="AppBundle\Content\MediaFactory" >
+            <deprecated>The "app.content.media_factory" is deprecated in favor of the AppBundle\Content\MediaFactory FQCN.</deprecated>
```

Comment:
maybe modify the calls of `app.content.media_factory` instead of add this deprecated message?
(the same comment for others deprecated messages)